<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowSelectDialog</class>
 <widget class="QDialog" name="WindowSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Window:</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="selectButton">
     <property name="text">
      <string>Click to select</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Handle:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="handleLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Title :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowSelectDialog</receiver>
   <slot>OnSelectWindowButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowSelectDialog</receiver>
   <slot>OnCancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowSelectDialog</receiver>
   <slot>OnConfirmButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSelectWindowButtonClicked()</slot>
  <slot>OnCancelButtonClicked()</slot>
  <slot>OnConfirmButtonClicked()</slot>
 </slots>
</ui>
